android/docs/engine/KMManager/applyKeyboardHeight.md

```diff
@@ -5,20 +5,28 @@ title: KMManager.applyKeyboardHeight()
 ## Summary
 The **applyKeyboardHeight()** method sets the height of the keyboard frame for 
 the device's current [screen orientation](https://developer.android.com/training/multiscreen/screensizes#TaskUseOriQuali) 
-(portrait vs landscape).
+(portrait vs landscape) or a specified orientation.
 
 ## Syntax
 ```java
 KMManager.applyKeyboardHeight(Context context, int height)
 ```
+or
+
+```java
+KMManager.applyKeyboardHeight(Context context, int height, int orientation)
+```
 
 ### Parameters
 
 `context`
 : The context
 
 `height`
-: The height of the keyboard frame in *density-independent pixels (dp)*
+: The height of the keyboard frame in *density-independent pixels (dp)*. Pass `KMManager.KeyboardHeight_Reset` to reset the keyboard height (for current orientation) to device-specific defaults.
```

Comment:
Can you also update the verbage about the reset button in?
https://github.com/keymanapp/keyman/blob/5f0f4b4f5b122f40ac1f02ca93a3ddb3cdef9bf6/android/docs/help/basic/config/adjust-keyboard-height.md?plain=1#L13-L16

Reply:
Done.